src/Native/Date.js

```diff
@@ -27,6 +28,12 @@ Elm.Native.Date.make = function(localRuntime) {
 		["Jan", "Feb", "Mar", "Apr", "May", "Jun",
 		 "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
 
+	currentDate = Task .asyncFunction (
```

Comment:
Thanks, this should read `var currentDate = ...`, so I missed the `var`.
Or do you have other concerns?